docs/scos/dev/example-modules.md

```diff
@@ -0,0 +1,66 @@
+
+## The difference between core and example modules
+
+**Core modules** provide generic, reusable functionality that benefits a wide range of projects. They offer a foundation for customization and serve as versatile building blocks.
+
+**Example Modules** in contrast, are designed for specific, one-off use-cases. Their functionality is tightly aligned with unique business requirements and isn't suitable for broad reuse. These modules serve as specialized, project-specific solutions.
+
+## Example module use-cases
+
+### SCOS: Showcase functionality
+
+SCOS uses example modules to fill the gap with concrete/example implementation over an abstract workflow (latter is the target of delivery), where the concrete/example implementation is never actually usable by any concrete customer or business use-case, thus the actual business logic is not part of the core (eg: Example picking strategy).
```

Comment:
Instead of the "example picking strategy", let's use another example that is better known

Reply:
OK, then let's use the React API example, because the Blog Example does not really fit the definition of "nonreuseable buisness logic" modules very well. All of the better example modules are new and mostly related to unified commerce.

My original idea was to add a list of modules, but this needs to be done after we fixed the readmes and license files for the old modules.